templates/jenkins/meta.yaml

```diff
@@ -0,0 +1,85 @@
+name: Jenkins
+description:
+  Jenkins is an open source automation server. It helps automate the parts of software development related to building, 
+  testing, and deploying, facilitating continuous integration, and continuous delivery. It is a server-based system that 
+  runs in servlet containers such as Apache Tomcat.
+instructions: null
+changeLog:
+  - date: 2024-10-06
+    description: First Release of Version 2.474
+links:
+  - label: Website
+    url: https://www.jenkins.io/
+  - label: Documentation
+    url: https://www.jenkins.io/doc/book/
+  - label: Github
+    url: https://github.com/jenkinsci/jenkins
+contributors:
+  - name: Ahson Shaikh
+    url: https://github.com/MuhammadAhsanDonuts
+schema:
+  type: object
+  required:
+    - appServiceName
+    - appServiceImage
+  properties:
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: jenkins_web
```

Comment:
use `jenkins` instead of `jenkins_web` - when we have only one service it doesn't make sense to add suffixes